Makefile

```diff
@@ -27,8 +27,12 @@ app-mac:
 
 .PHONY: backend
 backend:
+ifeq ($(OS), Windows_NT)
```

Comment:
On Windows maybe we get more than the `Windows_NT` value.
In fact, I have read that the `OS` may be available on Windows prior to Windows 7 (i.e. XP, NT). We only have to support >=7 AFAIU.
Have you checked maybe the `OSTYPE` variable which also exists on Linux?

A way to simplify the code (as to not repeat the whole command below) is to create a `SERVER_EXEC_EXT` variable, and assign it to .`exe` on Windows + leaving it empty otherwise. Then you use just one command with `... go build -o ./server${SERVER_EXE_EXT} ./cmd`.